README.md

```diff
@@ -21,6 +21,13 @@ Code. This extension uses [DVC](https://dvc.org/), an open-source data
 versioning and ML experiment management tool. No additional services or
 databases are required.
 
+🔔 Stay up-to-date with the latest product updates and tutorials by subscribing
+to the [DVC YouTube channel](https://www.youtube.com/@dvcorg8370/videos)!
+
+👉 [Subscribe now](https://www.youtube.com/@dvcorg8370/videos) to get notified
```

Comment:
```suggestion
👉 [Subscribe to our YouTube channel](https://www.youtube.com/@dvcorg8370/videos) to get notified
```

Reply:
```
🔔 Don't miss out on the latest tips and tricks for using DVC in your projects!
Subscribe to the DVC YouTube channel now to stay up-to-date with the latest
product updates and tutorials.

👉 [Subscribe now](https://www.youtube.com/@dvcorg8370/videos) to get notified
whenever we post new content. You'll get access to exclusive tutorials, demos,
and best practices that will help you get the most out of DVC in your workflow.
Join our community of data scientists and developers today! 💻🚀
```
copilot also suggested this when I told it to "update the statement to show that subscribing will provide value"

Reply:
If we just leave with just "subscribe now" they may think it mean :
1. more email 🙅🏽‍♂️, or
2. I'm already subscribed

Clarifying that it's youtube will create less resistance.

Reply:
Do you think we can drop the second line? Seems repetitive.

Reply:
or the first 🤷🏻

Reply:
I'm also unsure about having two separate lines that are taking you to the same youtube page 🤔

Reply:
Actually, you are so right @julieg18! I missed that the first line had a link.  I think just the first line is fine.  Dump the second line I think.